cloudhub-2/modules/ROOT/pages/ps-outbound-private-link.adoc

```diff
@@ -0,0 +1,113 @@
+.. https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html[Configure an endpoint service]
+.. In the VPC *Endpoint services*, click *Allow principals*
+... Add `arn:aws:iam::{accountId}:root` as principal
++
+The `{accountId}` is obtained in the first step step here
+. Gather this information for the API:
+.. Service Name: the endpoint service name
++
+For example, customer-hosted endpoint service: `com.amazonaws.vpce.us-east-1.vpce-svc-xxxxxxxxxxxxxxxx`
++
+For example, AWS hosted service: `com.amazonaws.us-west-2.s3`
+.. Service Region: the region where the VPC endpoint service is located
+.. Service Owner: the owner of the VPC endpoint service
+... Account id of the endpoint service 
+... “amazon”, if it’s an Amazon service
+.. Availability Zone Ids: the id of the availability zones where you wish to establish the private link. Make sure that the AZ Ids are the ones used by the private space (obtained in the first step here), and the VPC endpoint service is hosted in those AZ Ids.
+. Create a VPC endpoint in Cloudhub 2.0 via API
++
+[source,curl,linenums]
+----
+curl -XPOST https://anypoint.mulesoft.com/runtimefabric/api/{orgId}/privatespaces/{spaceId}/vpces -H "Authorization:$AUTHTOKEN"  -H "Content-Type:application/json"  -d '{
+	"name": "{any name}",
+	"serviceName": "{service name}",
+	"serviceRegion": "{service region}",
+	"serviceOwner": "{service owner}",
+	"azIds": [{aws-az-id1}, {aws-az-id2}]
+}‘
+----
++
+If the VPC endpoint is successfully created, the API returns a `vpceId`.
+. Get the VPC endpoint status in Cloudhub 2.0 using API
++
+[source,curl,linenums]
+----
+curl -XGET https://anypoint.mulesoft.com/runtimefabric/api/{orgId}/privatespaces/{spaceId}/vpces/{vpceid} -H "Authorization:$AUTHTOKEN" 
+----
++
+.. If successful, it returns the DNS names of the VPC endpoint
+.. It also returns the the provisioning status:
+... `Invalid`: VPCE creation failed
+... `Valid`: VPCE is being provisioned
+... `Available`
+... `PendingAcceptence`
+. Use this command to update a VPC Endpoint:
++
+[source,curl,linenums]
+----
+curl -XPATCH https://anypoint.mulesoft.com/runtimefabric/api/{orgId}/privatespaces/{spaceId}/vpces/{vpceId} -H "Authorization:$AUTHTOKEN"  -H "Content-Type:application/json"  -d '{
+	"name": "{any name}",
+	"serviceName": "{service name}",
+	"serviceRegion": "{service region}",
+	"serviceOwner": "{service owner}",
+	"azIds": [{aws-az-id1}, {aws-az-id2}]
+}‘
+----
++
+. Use this command to delete a VPC Endpoint:
+[source,curl,linenums]
+----
+curl -XDELETE https://anypoint.mulesoft.com/runtimefabric/api/{orgId}/privatespaces/{spaceId}/vpces/{vpceid} -H "Authorization:$AUTHTOKEN" 
+----
+
+
+== Limitations
+
+* This configuration supports interface endpoints only
+* This configuration doesn't support CloudHub VPCs or CloudHub 2.0 migrated private spaces
+* You can't perform cross-region validation properly done at the API level. If the endpoint service is invalid, the VPCE provisioning fails.
+
```

Comment:
We need a section on entitlements - each VPC endpoint will consume one of their network connection (in the new P&P) and VPN (in the legacy vcore p&p) entitlements.